logging/google/cloud/logging/handlers/GKEHandler.py

```diff
@@ -0,0 +1,56 @@
+# Copyright 2016 Google Inc. All Rights Reserved.
```

Comment:
(For both this file and `GAEHandler`) - Python modules *never* use case sensitive file names. Please use `app_engine.py` and `container_engine.py`.